lib/components/Input/text-field-with-steppers/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { ComponentProps, forwardRef } from "react";
+import Input from "../base";
+import React from "react";
+import clsx from "clsx";
+import {
+    LabelPairedPlusSmRegularIcon,
+    LabelPairedMinusSmRegularIcon,
+} from "@deriv/quill-icons";
+import "./text-field-with-steppers.scss";
+
+export interface TextFieldWithSteppersProps extends ComponentProps<typeof Input> {
+    unit?: string;
+    decimals?: number;
+    unitPlacement?: 'left' | 'right';
```

Comment:
can we use the generic type `TLeftOrRight`

Reply:
Yes, I didn't think there's already such type